### ISSUE TITLE
Does not work at all in V10

### ISSUE THREAD

**Issue by SLinxTheFox** [ElfFriend-DnD/foundryvtt-5eOGLCharacterSheet#38]
Just letting you know. I'm super grateful you keep this module working. It's the only character sheet i can stand. If i had the skills to take it over for you i would

**Comment by AndreCaseyScott** [comment]
I can confirm. Does not show up as a Character Sheet option in v10. Hope someone picks up this project, my players loved using it.

**Comment by Shepardnick** [comment]
Yes, my players and I have been using this after transferring from roll20 early this year. I find all the other available character sheets to be too spread out with their categories and information tabs. My players and I appreciate the work you've done on this module, and I hope it'll become available for v10.

**Comment by Hokage3211** [comment]
#40 in case anyone is needing this, check the releases of the repository for the pull request.

**Comment by mrmchughes** [comment]
I had this module working on Foundry Version 9, after updating to the latest patch of Version 10, the OGL sheet no longer appears as a sheet option. I have attempted deactivating every module but this sheet, and it still does not appear, so I think it is something with Version 10. Thank you for continuing the work on this, hope it can get working again!

**Comment by hedgehog8711** [comment]
I would love to see this working with the new version of foundry it looks like they have landed on a stable build if that helps for development. I hope they come back to this project and update it a lot of my group loves the old school sheets, and hates the new ones lol. Thank you devs in advance for your help.

**Comment by AndreCaseyScott** [comment]
From what I can tell, the code in master includes a fix for V10. I even saw a message from the author saying it's fixed for v10. Still doesn't show. 

The manifest.json still references the most latest release on github, which does not include the fix. If @akrigline will create a new release using the latest master branch, then the manifest.json can be updated and the fix applied. At least that's what I suspect at the moment.

**Comment by hedgehog8711** [comment]
Thank you for the reply, I took a look at the fix and must admit coding skills suck, I hope that we can get a new release soon. Thank you again for your reply. 
ThanksGreg
    On Saturday, January 7, 2023 at 08:07:01 PM EST, André Scott ***@***.***> wrote:  
 
 


From what I can tell, the code in master includes a fix for V10. I even saw a message from the author saying it's fixed for v10. Still doesn't show.

The manifest.json still references the most latest release on github, which does not include the fix. If @akrigline will create a new release using the latest master branch, then the manifest.json can be updated and the fix applied. At least that's what I suspect at the moment.

**Comment by mcabel** [comment]
does anyone know if the proposed changes in #40  fix the issues and allow running this on v10?
